### ISSUE TITLE
duros-controller should be restarted on changed adminToken

### ISSUE THREAD

**Issue by robertvolkmann** [metal-stack/gardener-extension-provider-metal#441]

**Comment by majst01** [comment]
The GEPM could even recreate the Duros Admin JWT Token on a regular basis, and then restart the duros-controller. The CA could be stored in the seed in a secret

**Comment by Gerrit91** [comment]
Maybe @ostempel wants to migrate the duros integration into a dedicated Gardener extension. In this case the controller would automatically be restarted when the deployment configuration changes. References https://github.com/metal-stack/duros-controller/issues/86.

**Comment by ostempel** [comment]
yes already spoke with @Gerrit91 about this. I will soon look into this
